### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by rasibley**
It's a start :-) I will provide more details in the internal documentation, there is a lot of internal workflow and details that should remain downstream probably, e.g. bkr job-submit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +14,24 @@ install it via pip and use it directly from your `PATH`.
...
+
+```bash
+$ git clone <kpet-db>
+```
 
+How to run it
+-------------
+To preview patch generated test cases:
 ```bash
-$ pip install --user -e .
```
Comment (agustinhenze) [inline README.md:18]: Did you remove it because you think it is redundant?

File: README.md [hunk 2]
```diff
@@ -12,10 +14,24 @@ install it via pip and use it directly from your `PATH`.
...
+$ kpet --db . run print-test-cases 001.patch
```
Comment (agustinhenze) [inline README.md:31]: I don't know if adds some value here, but patches can be remote urls as well.

**Review by agustinhenze — APPROVED** [review]
Thank you! I like to see how it is taking shape :) @rasibley. I have two comments but the PR can be merged

File: README.md (hunk 1 above)
Comment (rasibley) [reply README.md:18]: @agustinhenze yeah I thought it was a little redundant and was trying to simplify it, feel free to add it back or I can include it in the next version

File: README.md (hunk 2 above)
Comment (rasibley) [reply README.md:31]: @agustinhenze yes! definitely important to add that, I plan to update it with more details, feel free to merge or reject, and I can update it to look similar to the internal docs (sanitized version)

**Comment by agustinhenze** [comment]
OK, let's keep improving the doc in further PRs :)

**Pull request merged**